package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@artsy/reaction-force",
-  "version": "0.23.1",
+  "version": "0.0.0-development",
```

Comment:
Will this account for existing packages that have already been published to NPM? Does it determine from there that the existing release is `0.23.1` and go from there?

Reply:
Yes it should fetch the version number from the existing package on npm.

Reply:
When merging, the build on Circle will log the build evaluation and which commits trigger changes, as well as the new version number.

Reply:
Sweet, just wanted to double check 👍

Reply:
Also wanted to note it looks like this version number is being overwritten when I push a commit, but don't see where this is being set.